docs/scope.md

```diff
@@ -0,0 +1,53 @@
+
+# Definitions
+
+* **Showdown Supported (SS)**: A core series game who's metagame can be played on Showdown.
+  * Notably, this is every [core series game](https://bulbapedia.bulbagarden.net/wiki/Core_series#List_of_core_series_games) except Pokémon Legends: Arceus and Pokémon Legends: Z-A.  
+* **Base Expansion Version**: A .gba file built from an unmodified `master` branch of `pokeemerald-expansion`.  
+* **Vanilla Emerald Version**: A .gba file built from an unmodified `master` branch of pret's `pokeemerald`.
+
+# Guidelines
+
+A pull request meets the scope criteria if:
+* The feature does not belong to a category considered “not in scope” AND
+* The feature belongs to a category considered “in scope”
+
+## In Scope Categories
+
+1. **SS Species:** Adds Species that have appeared in a Showdown-supported title  
+2. **SS Moves:** Adds Moves and Move Animations that have appeared in a Showdown-supported title  
+3. **SS Abilities:** Adds Abilities that have appeared in a Showdown-supported title  
+4. **SS Items:** Adds Items that have appeared in a Showdown-supported title  
+5. **SS Gimmicks:** Adds Gimmicks that have appeared in a Showdown-supported title  
+6. **SS Battle Types:** Adds Special Battle Types that have appeared in a Showdown-supported title  
+7. **SS Battle Mechanics:** Adds mechanical battle changes that have appeared in a Showdown-supported title
+8. **Improve Battle AI:** Improve the Battle AI in a way that allows it to approach the skill and capability of a human competitive player  
+9. **Base Link Compatibility:** Link compatibility with base  
+10. **SS Overworld / Menu Updates:** Replicate overworld or menu changes from Showdown-supported Pokémon titles  
+11. **Speed Up:** Speed up the player experience of features found in base  
+12. **Compression:** Automatically compress assets  
+13. **Novel Experience:** Adds a novel experience included in another Showdown Supported title
+15. **Helper Features:** Eases the addition or inclusion of any of the aforementioned
+
+## Not In Scope Categories
+
+1. **Non-SS Species**: Adds Species that have NOT appeared in a Showdown-supported title  
+2. **Non-SS Moves**: Adds Moves and Move Animations that have NOT appeared in a Showdown-supported title  
+3. **Non-SS Abilities**: Adds Abilities that have NOT appeared in a Showdown-supported title  
+4. **Non-SS Items**: Adds Items that have NOT appeared in a Showdown-supported title  
+5. **Non-SS Gimmicks**: Adds Gimmicks that have NOT appeared in a Showdown-supported title  
+6. **Non-SS Battle Types**: Adds Special Battle Types that have NOT appeared in a Showdown-supported title  
+7. **Duplicate Feature UI**: Adds functionality that duplicates the core functionality of an existing vanilla feature
+8. **Vanilla Link Compatibility**: Link compatibility with vanilla  
+9. **External Program**: External programs
```

Comment:
We should probably specify that we mean external programs by those that don't automatically work as part of the `make` process.

Reply:
Agreed, will add once https://github.com/rh-hideout/pokeemerald-expansion/pull/5706#discussion_r1866265003 is addressed

Reply:
this implies that poryscript is in scope, whereas we explicitly voted that it wasn't

Reply:
Then how about "external executables or build tools actively maintained by a different team"?

Reply:
I'm not sure how that's different than just a blanket "external tools are outside of scope", are you trying to leave an opening for something specific?

Reply:
Mainly to allow us to have our own tools and not get lawyered by people saying that we technically have tools such as the teachable learnset helper

Reply:
Forgive me if I'm misunderstanding, but sure those are _internal_ tools?

Reply:
> Forgive me if I'm misunderstanding, but surely those are _internal_ tools?

...you got me there 😂
Alright, the current description for this makes sense then xD

Reply:
Are we good for approval and merging then lol

Reply:
what is an external tools and an internal? 

Can an external tool not become an internal tool?

Reply:
trainerproc was an external tool until it became an internal tool. Same with the teachable learnset tool

Reply:
Internal = expansion made for expansion use and thus expansion supported and included in our repo
External = supporting tool for decomp hacking that may or may not be expansion compatible but is maintained by others outside of our repo

> Can an external tool not become an internal tool?

Sure! But they are not in scope by default.

Reply:
I actual disagree here. Just because an we agreed Poryscript is out of scope doesn't mean other tools can't be in scope

Reply:
So you'd move it to "Discussion Required Categories" instead?

Reply:
Right, Idt a good tool should be blocked because we set a rule for it.

Reply:
9b06f7e4924604b8f69619009829ab0ba1dee437

if anybody disagrees, please propose an alternative